app/forms/dependant_form/monthly_income_form.rb

```diff
@@ -0,0 +1,33 @@
+module DependantForm
+  class MonthlyIncomeForm
+    include BaseForm
+    form_for Dependant
+
+    before_validation :clear_monthly_income
+
+    attr_accessor :has_income, :monthly_income
+
+    validate :income_presence
```

Comment:
Why doesn't this work:
```ruby
validates :has_income, presence: true
```
That would avoid using a custom method.

Reply:
I used the custom method because the error message required the dependants name within it e.g. Select yes if John receives any income